### PR TITLE
[WiP]: add Ukrainian language support

### PR DESCRIPTION
**Pull request by tutizaraz**
Hey, I'm super excited that this project exists. It's really important for the community, and we need to support as many languages as possible so more people can get into Nostr.

In this PR, I followed the contribution guide. If I broke any rules, I'd be happy to fix them. I encountered some issues in this PR, which I've described here -> #193.

I want to take a few days to review everything again and ensure the translation is as accessible as possible for Ukrainian speakers. I'll let you know when this PR is ready to be merged into the main branch.

/cc @erskingardner

### CODE REVIEW

File: src/routes/pages/uk/guides/astral.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (erskingardner) [inline src/routes/pages/uk/guides/astral.md:1]: You can remove this one. I don't think it's used anymore.

**Review by erskingardner — COMMENTED** [review]
Looks great so far. Let me know when you're ready for a final review

**Comment by tutizaraz** [comment]
@erskingardner why do we need both package-lock and pnpm-lock files? I guess we just need pnpm-lock

**Comment by erskingardner** [comment]
> @erskingardner why do we need both package-lock and pnpm-lock files? I guess we just need pnpm-lock

We don't. the package-lock probably snuck in there at some point.... thanks for removing.

**Comment by tutizaraz** [comment]
@erskingardner this PR is ready for review

**Review by erskingardner — APPROVED** [review]

**Pull request merged**